cmd/akash/cmd/genaccounts.go

```diff
@@ -158,14 +158,14 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 
 			bankGenStateBz, err := cdc.MarshalJSON(bankGenState)
 			if err != nil {
-				return errors.Errorf("failed to marshal bank genesis state: %w", err)
+				return errors.Errorf("failed to marshal bank genesis state: %v", err)
 			}
 
 			appState[banktypes.ModuleName] = bankGenStateBz
 
 			appStateJSON, err := json.Marshal(appState)
 			if err != nil {
-				return errors.Errorf("failed to marshal application genesis state: %w", err)
+				return errors.Errorf("failed to marshal application genesis state: %v", err)
```

Comment:
`%w` is for [wrapping errors](https://blog.golang.org/go1.13-errors)

Reply:
Thanks, aware of this. `make test-full` was throwing errors/warnings telling me that doing `%w` calls in `Errorf` is not recommended which is why I changed them. If you would like me to change them back I'm happy to.

Reply:
Marking resolved per our conversation at standup.